sets.json

```diff
@@ -445,13 +445,13 @@
 		"card_set_type_code": "villain"
 	},
 	{
-		"code": "collector1",
-		"name": "Collector 1",
+		"code": "infiltrate_the_Museum",
+		"name": "Infiltrate the Museum",
 		"card_set_type_code": "villain"
 	},
 	{
-		"code": "collector2",
-		"name": "Collector 2",
+		"code": "escape_the_Museum",
```

Comment:
FYI, the code pattern is lowercase snake_case so I think we'd want `infiltrate_the_museum` and `escape_the_museum` (make the M's lowercase m's)